include/picongpu/param/incidentField.param

```diff
@@ -38,10 +38,10 @@
  * `OFFSET` that controls inward offset of the generating surface relative to global domain. For example:
  *
  * @code{.cpp}
- * using XMin = profiles::Free< UserFunctorIncidentE >;
+ * using XMin = profiles::Free< UserFunctorIncidentE,  UserFunctorIncidentB>;

```

Comment:
Two spaces after the comma.
Could you please fix the code style of the following lines too, there should be no space before and after `<,>`